src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/enchanting/AutoEnchanter.java

```diff
@@ -117,7 +122,13 @@ protected MachineRecipe findNextRecipe(BlockMenu menu) {
 
     private boolean isEnchantable(ItemStack item) {
         SlimefunItem sfItem = null;
-
+        if (item != null && item.hasItemMeta() && item.getItemMeta().hasLore()) {
+            for (String lore : cantEnchantLores.getValue()) {
```

Comment:
Same here, assign getItemMeta() to a variable as well as getLore()

Reply:
edited